developer_docs/installation.md

```diff
@@ -80,3 +80,10 @@ name, you can also set it using this variable. I.e.:
 For more information on using a custom domain, see this documentation link:
 
 http://docs.aws.amazon.com/AmazonS3/latest/dev/VirtualHosting.html#VirtualHostingCustomURLs
+
```

Comment:
I think it makes sense to link to this section the [Manual Installation](https://github.com/processing/p5.js-web-editor/blob/6a6eeb21ef386c30a81ca6172b4557a4c92f5223/developer_docs/installation.md#manual-installation) section, step 9, and the [Docker Installation](https://github.com/processing/p5.js-web-editor/blob/6a6eeb21ef386c30a81ca6172b4557a4c92f5223/developer_docs/installation.md#docker-installation) section, step 6, since it's necessary to fetch the examples.